vsts/test-release-nuget.yaml

```diff
@@ -0,0 +1,265 @@
+name: $(BuildID)_$(BuildDefinitionName)_$(SourceBranchName)_$(Date:yyyyMMdd)$(Rev:.r)
+
+resources:
+- repo: self
+  clean: true
+phases:
+
+### Linux build ###
+- phase: LINUX
+  displayName: Linux
+
+  condition: succeeded()
+  queue:
+    name: Hosted Ubuntu 1604
+    timeoutInMinutes: 180
+
+  steps:
+
+  - task: DownloadBuildArtifacts@0
+    inputs:
+      buildType: 'specific'
+      project: 'f9b79625-2860-4d92-a4ee-57b03fabfd10'
```

Comment:
Where do we get these values from?

Reply:
They are generated by the build yaml snippet builder.  I don't see them displayed on any of the pages so I added comments for their display names